docs/SUMMARY.md

```diff
@@ -5,13 +5,20 @@
 * [About MakeCode](/about)
 * [Blog](/blog)
 * [Contact Us](/contact)
+* [Accessibility](/accessibility)
+    * [Keyboard Navigation](/accessibility/keyboard-navigation)
```

Comment:
For editor accessibility features, make it a one pager. and use something like this format: 
https://code.visualstudio.com/docs/editor/accessibility